api/common_stuff.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"google.golang.org/api/googleapi"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I found three different naming of  k8s.io/apimachinery/pkg/apis/meta/v1, it would be fine to use one common name insted of metav1, v1meta and meta_v1.

Reply:
I would rather do this in a subsequent PR as I think this is probably a broader problem.